src/app-layer-htp.c

```diff
@@ -488,21 +492,6 @@ void AppLayerHtpEnableResponseBodyCallback(void)
     SCReturn;
 }
 
-/**
```

Comment:
why is this logic removed?

Reply:
this type of change should be explained in the commit message

Reply:
This is just dead code removal :

`AppLayerHtpEnableRequestBodyCallback` is called anyways by `AppLayerHtpNeedMultipartHeader `'s only caller (`AppLayerHtpNeedFileInspection`)

And `HTP_REQUIRE_REQUEST_MULTIPART` is unused.

What more would you want in the commit message ?